services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -85,9 +109,9 @@ def _get_spooled_file_size(file_io: IO) -> int:
 @cancel_on_disconnect
 async def upload_file(
     request: Request,
-    file: UploadFile = FileParam(...),
-    content_length: str | None = Header(None),
-    user_id: int = Depends(get_current_user_id),
+    file: Annotated[UploadFile, FileParam(...)],
+    user_id: Annotated[int, Depends(get_current_user_id)],
+    content_length: str | None = Header(None),  # noqa: B008
```

Comment:
what is this comment for? don't they have a readable version?

Reply:
Those are new ruff comments (see PR from @sanderegg )

Reply:
yep. I'm "slowly" refining. it's going to become better.